libs/wingsdk/src/std/array.ts

```diff
@@ -125,6 +145,19 @@ export class Array {
     searchElement;
     throw new Error("Macro");
   }
+
+  /**
+   * Returns a sorted copy of the array.
+   *
+   * @macro ((arr, args) => { return args && args[0] ? [...(arr)].sort(args[0]) : [...(arr)].sort() })($self$, $args$)
```

Comment:
```suggestion
   * @macro ($self$.sort($args$ ?? undefined))
```

- $args$ are the arguments passed as-is, not necessarily an array
- We don't actually need the wrapper function here, just calling sort is fine
- I think JS's sort handles `undefined` so it can just be passed in. adding `?? undefined` is just in case args is null which I think is possible

The other macros here can be changed similarly